changelog/update-rename-payment-activity-singular

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: update
+
+Update: Fixed spelling of `Payments` to `Payment`. Changes are behind a feature flag.
```

Comment:
IMO, since code has not been released yet, any modification to it does not need a visible changelog, so this should be just a comment that will not generate a changelog entry.

Reply:
I agree, this change should be covered by changelog item for the feature (behind flag).

Reply:
@shendy-a8c  - Sorry about a silly question here. When we say `this should be just a comment`,  can you give me an example? Do I remove this changelog file, or probably say `Comment: This related to PR #--- and contains some minor spelling corrections on top of it`. I want to be sure I am doing this correctly.

Reply:
Sorry for being ambiguous.

> or probably say Comment: This related to PR #--- and contains some minor spelling corrections on top of it.

☝️ This is what I meant.

Here's how I usually did it:
```
% npm run changelog

> woocommerce-payments@7.4.0 changelog
> ./vendor/bin/changelogger add

Name your changelog file [default: add-8519-reporting-api-to-component] > abc
Significance of the change, in the style of semantic versioning.
  [patch] Backwards-compatible bug fixes.
  [minor] Added (or deprecated) functionality in a backwards-compatible manner.
  [major] Broke backwards compatibility in some way.
 > patch
Type of change.
  [add   ] Add
  [fix   ] Fix
  [update] Update
  [dev   ] Dev
 > dev
Changelog entry. May be left empty if this change is particularly insignificant.
 > 
You omitted the changelog entry, which is fine. But please comment as to why no entry is needed.
 > type test comment here
```

Reply:
Ah! Thanks for explaining this in detail, @shendy-a8c 🙏! I never reached the comment prompt on the npm run changelog :D . It is very clear now!